tools/osbuilder/rootfs-builder/init_trusted_storage.sh

```diff
@@ -0,0 +1,51 @@
+#!/bin/bash
+#
```

Comment:
Some random thoughts:

- If we were _really_ paranoid, this script could check to ensure it was being called by the agent process and `die()` (see below) if not.
- This script isn't idempotent. That _shouldn't_ be a problem, but with my paranoia hat on again it might be best if it is!